src/misc1.c

```diff
@@ -4470,6 +4470,19 @@ remove_tail(char_u *p, char_u *pend, char_u *name)
     return pend;
 }
 
+    void
+vim_unsetenv(char_u *var)
+{
+#ifdef _WIN32
+    mch_setenv((char *)var, "", 0);
+#elif defined(HAVE_UNSETENV)
+    unsetenv((char *)var);
+#else
+    EMSG(_("E319: Sorry, the command is not available in this version"));
```

Comment:
Do not copy message text over, if needed place that in globals.h near other error messages and use in two places. Though I would be against this message: *command* is available, specific part of command’s functionality is missing and it deserves its own message.